Kong/docs.konghq.com
# Add KongConsumerGroup documentation for KIC

### Description

Add KongConsumerGroup to the KIC CRD and plugin documentation. The reference is untouched, since that will be handled separately by an autodoc system.
 
closes https://github.com/Kong/kubernetes-ingress-controller/issues/4394

### Testing instructions

Netlify link:

https://deploy-preview-5865--kongdocs.netlify.app/kubernetes-ingress-controller/latest/guides/using-kongplugin-resource/
https://deploy-preview-5865--kongdocs.netlify.app/kubernetes-ingress-controller/latest/concepts/custom-resources/

However the sections are invisible due to the version not being out yet. I forget, do we have a way to fake that without actually adding the standard version content (editing the nav and such)?

Partially practical tested to verify the resources and patches. Complete testing is pending full implementation in KIC (https://github.com/Kong/kubernetes-ingress-controller/issues/3728) and a gateway fix.

### Checklist 


Unsure if we're doing a staging branch for 2.11. This content is all version gated and could go direct to main.
